commands/info/command.go

```diff
@@ -64,11 +67,23 @@ type infoResult struct {
 	infoResponse     *clientinfo.GetInfoOK
 }
 
-func init() {
-	RootCommand.AddCommand(InfoCommand)
+// validatePreconditions simply returns nil, as the command should work under
+// all conditions.
+func validatePreconditions(args infoArguments) error {
+	return nil
 }
 
-// printInfo prints some information on the current user and configuration
+// printValidation prints if there is anything missing from user input or config.
+func printValidation(cmd *cobra.Command, extraArgs []string) {
+	args := defaultInfoArguments()
+	err := validatePreconditions(args)
+
+	if err != nil {
+		errors.HandleCommonErrors(err)
+	}
+}
+
```

Comment:
Adding `validatePreconditions` and `printValidation` as they are in place normally, to explicitly remind us here that this command in fact does not have any preconditions to be met.